CL/cl_ext.h

```diff
@@ -1776,21 +1903,12 @@ clEnqueueReleaseExternalMemObjectsKHR(
 #define CL_KHR_EXTERNAL_MEMORY_DMA_BUF_EXTENSION_NAME \
     "cl_khr_external_memory_dma_buf"
 
-/* cl_external_memory_handle_type_khr */
```

Comment:
`cl_khr_external_memory_dx` has dissapeared after copying over `cl_ext.h` wholesale from the `gen_headers.py` output - However it looks like this was intended https://github.com/KhronosGroup/OpenCL-Docs/pull/1160